### PR TITLE
Personal XD75 keymap added.

### PR DESCRIPTION
**Pull request by BramVer**
Nothing fancy, just a no-hassle personal keymap for the XD75 keyboard.

### CODE REVIEW

File: keyboards/xd75/keymaps/bramver/config.h [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/xd75/keymaps/bramver/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/xd75/keymaps/bramver/config.h [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+#define CONFIG_USER_H
+
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/xd75/keymaps/bramver/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.

File: keyboards/xd75/keymaps/bramver/keymap.c [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "xd75.h"
```
Comment (drashna) [inline keyboards/xd75/keymaps/bramver/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead?

File: keyboards/xd75/keymaps/bramver/rules.mk [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Build options
+BACKLIGHT_ENABLE = yes		# Switch LEDs
+MOUSEKEY_ENABLE = yes			# Emulates mouse key using keypresses
+RGBLIGHT_ENABLE = yes 		# LED strips
+EXTRAKEY_ENABLE = yes 		# Use system and audio control key codes
+TAP_DANCE_ENABLE = no 		# Use multi-tap features
+AUDIO_ENABLE = no  				# Audio stuff
+UNICODEMAP_ENABLE = yes  	# Emojify me pls
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/xd75/keymaps/bramver/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/xd75/keymaps/bramver/config.h (hunk 1 above)
Comment (BramVer) [reply keyboards/xd75/keymaps/bramver/config.h]: No problem, don't know any C so I just adapted other people's way of doing things :)

Comment (drashna) [reply keyboards/xd75/keymaps/bramver/config.h]: Not a problem. And we've recently converted over to this convention, as it is simpler.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**